test/benchmarks.cpp

```diff
@@ -413,6 +413,12 @@ TEST_CASE("test_qft_permutation_round_trip_entangled", "[qft]")
         true, false, testEngineType == QINTERFACE_QUNIT);
 }
 
+TEST_CASE("test_qft_superposition_one_way", "[qft]")
```

Comment:
More test cases for all of the newly added gates, please.

Reply:
PhaseRootN variants plug right in as a common case of other gates and should be well-covered as part of the QFT. (This was basically the point, of that.) However, I'll add new tests for SqrtX/Y/H, which are not covered.